zeebe/exporters/operate-exporter/pom.xml

```diff
@@ -24,6 +24,11 @@
 
   <dependencies>
 
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-core</artifactId>
```

Comment:
We have this dependency for `StringUtils` class, right? Can we may be extend our own utility classes? Would be nice not to have Spring dependency on this module.

Reply:
@sdorokhova I do not use this dependency, but I added it because the CI fails without it, I do not understand why:
<img width="694" alt="image" src="https://github.com/user-attachments/assets/35402f1a-9cb2-4521-a1df-0897ec4db578">

Reply:
I will merge like this, to have the CI pass, as this dependency can be removed later.